store/db.go

```diff
@@ -104,6 +104,10 @@ func (db *DB) DoTx(fns ...txfunc) error {
 		return err
 	}
 	defer func() {
+		if r := recover(); r != nil {
```

Comment:
I noticed that, if a called function panics, the deferred function was going to call a tx.Commit (as on panic err is nil), with the possibility to commit an incomplete transaction, instead of doing rollbacking or returning an error (that will rollback on the next db.Close).

Thinking more about it, I reworked it to just not use a deferred function (as there's only one error path).